### PR TITLE
[libxml2.rb] enhancement

### PR DESCRIPTION
**Pull request by lovedheart**
libxml2 enhancement build with python and zlib

### CODE REVIEW

File: packages/libxml2.rb [hunk 1]
```diff
@@ -27,9 +17,9 @@ def self.build
       "--enable-shared",
       "--disable-static",
       "--with-pic",
-      "--without-python",
+      "--with-python",
       "--without-lzma",
```
Comment (uberhacker) [inline packages/libxml2.rb]: Why not `--with-lzma` also?

File: packages/libxml2.rb [hunk 2]
```diff
@@ -6,19 +6,9 @@ class Libxml2 < Package
   version '2.9.7'
```
Comment (uberhacker) [inline packages/libxml2.rb]: You should get in the habit of bumping the version whenever you make changes.

File: packages/libxml2.rb [hunk 3]
```diff
@@ -6,19 +6,9 @@ class Libxml2 < Package
...
-  })
-  binary_sha256 ({
-    aarch64: '26c83a2954c6180c3353d498bf6271216b9c12e11a7dc5f7207df9dc93c2b76a',
-     armv7l: '26c83a2954c6180c3353d498bf6271216b9c12e11a7dc5f7207df9dc93c2b76a',
-       i686: '650adc4a7aedbd6e9db24dff7d747ee3a9c87366b1d4aa012464867ebff15151',
-     x86_64: '2f55a258847137e155f029392119906803f5a4e361aa137235a6e79c5470716e',
-  })
+ 
+  depends_on 'python27'
+  depends_on 'zlibpkg'
```
Comment (uberhacker) [inline packages/libxml2.rb]: depends_on 'lzma'?

Comment (lovedheart) [reply packages/libxml2.rb]: It fails to build with lzma. Currently I can not fix the errors

**Pull request merged**